lib/api/calendars.ts

```diff
@@ -51,3 +51,17 @@ export const modifyEvent = (client: DAVClient) => async (eventPath: string, even
     body: translate(event)
   });
 }
+
+export const changeParticipation = (client: DAVClient) => async (eventPath: string, mailAttendee: string, action: string, event: CalendarEventObject): Promise<string> => {
```

Comment:
The arguments are long enough to be converted into an object argument.

Reply:
all the arguments is required to be passed in this case ! what we should do

Reply:
Convert it to an object argument.

```
export interface ChangeParticipationOptions {
  eventPath: string;
  attendeeEmail: string;
  action: string;
  event: CalendarEventObject;
}

export const changeParticipation = (client: DAVClient) => async ({ eventPath, mailAttendee, action, event }: ChangeParticipationOptions): Promise<string> => {
```

You can read https://github.com/ryanmcdermott/clean-code-javascript#function-arguments-2-or-fewer-ideally for more information about this rule.